engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -118,6 +118,11 @@ abstract class CromwellRootActor(terminator: CromwellTerminator,
     .props(CopyWorkflowLogsActor.props(serviceRegistryActor, ioActor)),
     "WorkflowLogCopyRouter")
 
+  //Call-caching config validation
+  lazy val callCachingConfig = config.getConfig("call-caching")
+  val callCachingEnabled = callCachingConfig.getBoolean("enabled")
+  val callInvalidateBadCacheResults = callCachingConfig.getBoolean("invalidate-bad-cache-results")
```

Comment:
These should both be `lazy val` like `callCachingConfig`, otherwise their definition as a `val` is immediately de-lazying the `lazy val`.